committeeoversightapp/models.py

```diff
@@ -72,23 +72,40 @@ class StaticPage(Page):
 
 
 class LandingPage(Page):
-    body = StreamField([
-        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
-        ('paragraph', blocks.RichTextBlock()),
-        ('image', ImageChooserBlock()),
-        ('button',
-         blocks.StructBlock([
-            ('button_text', blocks.CharBlock()),
-            ('button_link', blocks.URLBlock())
-            ],
-         icon='site'))
-    ])
+    body = RichTextField()
 
     # Editor configuration
     content_panels = Page.content_panels + [
-        StreamFieldPanel('body'),
+        FieldPanel('body'),
     ]
 
+    def get_context(self, request):
+        context = super(LandingPage, self).get_context(request)
+        context['house_committees'] = Organization.objects.all().filter(
+                                    classification='committee'
+                                ).filter(
+                                    parent_id__name__in=['United States House of Representatives']
+                                )
+        context['senate_committees'] = Organization.objects.all().filter(
+                                    classification='committee'
+                                ).filter(
+                                    parent_id__name__in=['United States Senate']
+                                )
+
+        committees = context['house_committees'] | context['senate_committees']
+
+        # Since Wagtail strips slashes out of urls but OCD prefixes their orgs
+        # with 'ocd-organization/', we need to do some cleaning. The cleaned_id
+        # of each committee should be of the form 'committee-<committee #>'
+        for committee in committees:
+            if not committee.extras.get('cleaned_id'):
```

Comment:
It could be derived from the committee IDs, which are already persistent—do you think this would be better as a template tag?

Reply:
i don't think a template tag would necessarily be appropriate, because we will want to use `cleaned_id` to route requests (i.e., get right object) to the committee detail page, in addition to rendering urls in the template. but i do think it makes sense to put this code somewhere it's easy to access and reusable. 

similarly to https://github.com/datamade/committee-oversight/pull/82#discussion_r326286522, i generally define values derived from a model's attributes as properties on the implicated model. do you think that could work here?

Reply:
Wouldn't that have to be on the `Organization` model, which is imported by OCD?

Reply:
adding a property would change the python api for the model, but it need not change the model at the database level. luckily, django offers [proxy models](https://docs.djangoproject.com/en/2.2/topics/db/models/#proxy-models) for this exact use case.

Reply:
Wow interesting!! This is exactly what we need.

Reply:
one thing to note, though, is that you can't use properties to filter querysets, so you can't do something like `Organization.objects.get(cleaned_id=blah)`, since `get` performs a database query, and we aren't storing `cleaned_id`. instead, you have to do the filtering in python. this can be ok – if your data isn't too big. about how many organizations will we be dealing with?

Reply:
Ah sorry missed this comment. There are around 40 organizations, so I think this will be fine.